testenvironments.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Is this file needed?

Reply:
The file allows execution of unit test directly from Visual Studion in WSL when developing on Windows, see the documentation here: https://learn.microsoft.com/en-us/visualstudio/test/remote-testing?view=vs-2022

Reply:
Alright